config/samples/certificate.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: cert-manager.io/v1alpha2
 kind: Issuer
 metadata:
-  name: harbor-sample
+  name: sample-sample
```

Comment:
Is this intentional? Previous name seemed better

Reply:
Was an old files that I just removed